src/app/App.tsx

```diff
@@ -0,0 +1,122 @@
+import { LayoutLogin } from 'features/netflixClone'
```

Comment:
bad imports. They should have 2 parts.
```
import { LayoutLogin, LayoutHome, LayoutRegistration } from 'features/netflixClone'
```

add additional index.ts inside `features/netflixClone`